From 845b60847743e58d2bf4fe4c9891d02d5b18736b Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
Date: Wed, 12 Oct 2016 22:52:51 -0700
Subject: [PATCH 9/9] config.h: Add missing include of stdint.h for uint64_t
 usage

Regressed-in: trunk r270692
Regressed-in: d2d1ea9d75dfc4f55540f7e3cf940c6a1d6674cc
Signed-off-by: Jeremy Huddleston Sequoia <jeremyhu@apple.com>
CC: Asiri Rathnayake <asiri.rathnayake@arm.com>
---
 src/config.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/config.h b/src/config.h
index 1f48163..3a67325 100644
--- a/src/config.h
+++ b/src/config.h
@@ -16,6 +16,7 @@
 
 #include <assert.h>
 #include <stdio.h>
+#include <stdint.h>
 #include <stdlib.h>
 
 // Define static_assert() unless already defined by compiler.
-- 
2.10.1

